Assets/Scripts/Menu/HighScores.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using UnityEngine;
+using TMPro;
+
+public class HighScores : MonoBehaviour
+{
+    public TMP_Text highScoreText;
+    // Start is called before the first frame update
+    void Start()
+    {
+        highScoreText = GetComponent<TMP_Text>();
+        if (PlayerPrefs.HasKey("TopScores"))
+        {
+            String[] topScores = PlayerPrefs.GetString("TopScores").Split("/n");
+            string temp = "";
+            for (int i = 0; i < 5; i++)
+            {
+                temp = temp + (i+1) + ". " + topScores[i] + "\n";
```

Comment:
Refer to constants file for string formatting

Reply:
what do you mean? This string is just generated for top scores

Reply:
Similar to this
![image](https://user-images.githubusercontent.com/55724149/179372327-9112d238-ec81-41c0-a2de-e0cd547657ca.png)
![image](https://user-images.githubusercontent.com/55724149/179372308-f6a32aeb-a4ba-4d84-9cc7-77e3dcb0a6e5.png)

Reply:
i dont think i can do this with the way im setting the text, im adding text to a string in a for loop

Reply:
I dont think this is neccesary because if we wanted to increase the number it would just require us to increase the size of the TopScores in playerprefs